clients/src/main/java/org/apache/kafka/common/requests/DescribeClientQuotasResponse.java

```diff
@@ -66,10 +66,11 @@ public DescribeClientQuotasResponse(Map<ClientQuotaEntity, Map<String, Double>>
     }
 
     public DescribeClientQuotasResponse(int throttleTimeMs, Throwable e) {
+        ApiError apiError = ApiError.fromThrowable(e);
```

Comment:
Should the same thing happen in `public AlterClientQuotasResponse(Collection<ClientQuotaEntity> entities, int throttleTimeMs, Throwable e)` for consistency?  And also maybe in `IncrementalAlterConfigsResponse` (`public static IncrementalAlterConfigsResponseData toResponseData`)?

Reply:
@rondagostino Thanks for the review. Updated `AlterClientQuotasResponse`. I think IncrementalAlterConfigsResponse is ok since it uses ApiError, we create the error with the right message when converting from Throwable. Can you just verify that the method with ApiError was the one you meant? Thanks,